### PR TITLE
fixes #10528 - lookup value should allow false and 0 values

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: app/models/lookup_value.rb [hunk 1]
```diff
@@ -24,6 +24,10 @@ class LookupValue < ActiveRecord::Base
   scoped_search :on => :match, :complete_value => true
   scoped_search :in => :lookup_key, :on => :key, :rename => :lookup_key, :complete_value => true
 
+  def value_present?
+    !!value.nil?
```
Comment (domcleal) [inline app/models/lookup_value.rb]: add an error?

Comment (unorthodoxgeek) [inline app/models/lookup_value.rb]: yeah, forgot how validators work. that's why tests failed, pushing fix in a
few minutes, once tests pass on my machine

Comment (unorthodoxgeek) [inline app/models/lookup_value.rb]: also added a test which I should have added before, and would have caught this silly error of mine :smile:

**Comment by domcleal** [comment]
Merged as ee6fc20, thanks @unorthodoxgeek.

**Pull request closed without merging**